internal/transactions/send.go

```diff
@@ -60,8 +60,15 @@ func send(
 	state *flowkit.State,
 ) (command.Result, error) {
 	codeFilename := args[0]
-
-	signer, err := state.Accounts().ByName(sendFlags.Signer)
+	// if sendFlags.Signer != emulator-account, this means that user has specified a signer in command flag
+	// if user has not specified signer in flag, we will use service account of default emulator as specified
+	// in flow.json
+	transactionSigner := sendFlags.Signer
+	if sendFlags.Signer == "emulator-account" {
```

Comment:
Checks if signer flag value is "emulator-account" (if it is that means that user did not specify signer in flag, and we will use the emulator service account defined in emulator section of flow.json